src/include/daos/pool_map.h

```diff
@@ -64,6 +76,13 @@ struct pool_component {
 	uint32_t		co_ver;
 	/** failure sequence */
 	uint32_t		co_fseq;
+	/**
+	 * version it's been EXCLUDE_OUT (when status set to
+	 * PO_COMP_ST_DOWNOUT).
+	 */
+	uint32_t		co_out_ver;
+	/** flags, see enum pool_component_flags */
+	uint32_t		co_flags;
```

Comment:
I'd rather just store the previous status, instead of creating a new flag.

Reply:
I was thinking that if store the previous status, will use up all the added 16B, here we only use 1 bit actually so probably some leftover can be useful later if we want to add something else.